### PR TITLE
Aerodrome strategy AMO deployment #006

### PR DESCRIPTION
**Pull request by sparrowDom**
## Contracts
| Contract | Address |
| - | - |
| AerodromeAMOStrategyProxy | [0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af](https://basescan.org/address/0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af) |
| AerodromeAMOStrategy (implementation) | [0xcDD21c5544A5B07fab409284cEE6c6097091B589](https://basescan.org/address/0xcDD21c5544A5B07fab409284cEE6c6097091B589) |

## Governance
```
Execute the following actions using guardian safe:  [
  {
    sig: 'claimGovernance()',
    args: [],
    to: '0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af',
    data: '0x5d36b190',
    value: '0'
  },
  {
    sig: 'approveStrategy(address)',
    args: [ '0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af' ],
    to: '0x98a0CbeF61bD2D21435f433bE4CD42B56B38CC93',
    data: '0x3b8ae397000000000000000000000000f611cc500eee7e4e4763a05fe623e2363c86d2af',
    value: '0'
  },
  {
    sig: 'setStrategistAddr(address)',
    args: [ '0x28bce2eE5775B652D92bB7c2891A89F036619703' ],
    to: '0x98a0CbeF61bD2D21435f433bE4CD42B56B38CC93',
    data: '0x773540b300000000000000000000000028bce2ee5775b652d92bb7c2891a89f036619703',
    value: '0'
  },
  {
    sig: 'addStrategyToMintWhitelist(address)',
    args: [ '0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af' ],
    to: '0x98a0CbeF61bD2D21435f433bE4CD42B56B38CC93',
    data: '0x1816dd4a000000000000000000000000f611cc500eee7e4e4763a05fe623e2363c86d2af',
    value: '0'
  },
  {
    sig: 'setHarvesterAddress(address)',
    args: [ '0x28bce2eE5775B652D92bB7c2891A89F036619703' ],
    to: '0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af',
    data: '0xc2e1e3f400000000000000000000000028bce2ee5775b652d92bb7c2891a89f036619703',
    value: '0'
  }
]

```

## Deploy checklist

Two reviewers complete the following checklist:

```
- [ ] All deployed contracts are listed in the deploy PR's description
- [ ] Deployed contract's verified code (and all dependencies) match the code in master
- [ ] The transactions that interacted with the newly deployed contract match the deploy script.
- [ ] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
```

### CODE REVIEW

**Comment by DanielVF** [comment]
- [x] All deployed contracts are listed in the deploy PR's description
- [x] Deployed contract's verified code (and all dependencies) match the code in master
- [x] The transactions that interacted with the newly deployed contract match the deploy script.
- [x] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal

**Review by DanielVF — DISMISSED** [review]

File: contracts/deployments/base/.migrations.json [hunk 1]
```diff
@@ -4,6 +4,8 @@
   "003_base_vault_and_token": 1722429523,
   "004_super_oeth": 1723219797,
   "005_mutlisig_harvester": 1723220782,
+  "006_base_amo_strategy": 1724882452,
   "007_bridged_woeth_strategy": 1724168481,
-  "008_oethb_zapper": 1724651792
+  "008_oethb_zapper": 1724651792,
+  "006_base_amo_strategy": 1724882095
```
Comment (shahthepro) [inline contracts/deployments/base/.migrations.json:10]: nit: Duplicate entry in the file. I believe you added one manually, can you remove it?

**Review by shahthepro — DISMISSED** [review]
- [x] All deployed contracts are listed in the deploy PR's description
- [x] Deployed contract's verified code (and all dependencies) match the code in master
- [x] The transactions that interacted with the newly deployed contract match the deploy script.
- [x] Governance proposal matches the deploy script
- [ ] Smoke tests pass after fork test execution of the governance proposal
 
### Code comparison
```
sol2uml diff 0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af .,node_modules --network base
Compared the "undefined" file for the "AerodromeAMOStrategyProxy" contract with address 0xF611cC500eEE7E4e4763A05FE623E2363c86d2Af on base
to local files under folders ".,node_modules"

match   @openzeppelin/contracts/utils/Address.sol
match   contracts/governance/Governable.sol
match   contracts/proxies/InitializeGovernedUpgradeabilityProxy.sol
match   contracts/proxies/Proxies.sol
```
```
sol2uml diff 0xcDD21c5544A5B07fab409284cEE6c6097091B589 .,node_modules --network base
Compared the "undefined" file for the "AerodromeAMOStrategy" contract with address 0xcDD21c5544A5B07fab409284cEE6c6097091B589 on base
to local files under folders ".,node_modules"

match   @openzeppelin/contracts/token/ERC20/IERC20.sol
match   @openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol
match   @openzeppelin/contracts/utils/Address.sol
match   @openzeppelin/contracts/utils/math/Math.sol
match   @openzeppelin/contracts/utils/math/SafeCast.sol
match   @openzeppelin/contracts/utils/math/SafeMath.sol
match   contracts/governance/Governable.sol
match   contracts/interfaces/aerodrome/ICLGauge.sol
match   contracts/interfaces/aerodrome/ICLPool.sol
match   contracts/interfaces/aerodrome/INonfungiblePositionManager.sol
match   contracts/interfaces/aerodrome/ISugarHelper.sol
match   contracts/interfaces/aerodrome/ISwapRouter.sol
match   contracts/interfaces/IBasicToken.sol
match   contracts/interfaces/IStrategy.sol
match   contracts/interfaces/IVault.sol
match   contracts/strategies/aerodrome/AerodromeAMOStrategy.sol
match   contracts/token/OUSD.sol
match   contracts/utils/Helpers.sol
match   contracts/utils/Initializable.sol
match   contracts/utils/InitializableAbstractStrategy.sol
match   contracts/utils/InitializableERC20Detailed.sol
match   contracts/utils/StableMath.sol
match   contracts/vault/VaultStorage.sol
```

File: contracts/deployments/base/.migrations.json (hunk 1 above)
Comment (shahthepro) [reply contracts/deployments/base/.migrations.json]: I pushed a fix for this here: https://github.com/OriginProtocol/origin-dollar/pull/2206/commits/0bfdebca12e34b07f9a432d25cce700e143e4baf

Comment (sparrowDom) [reply contracts/deployments/base/.migrations.json]: tnx

**Pull request merged**